MapboxCoreNavigation/RouteController.swift

```diff
@@ -262,13 +262,33 @@ open class RouteController: NSObject {
      */
     var rawLocation: CLLocation?
     
+    struct PreviousLocationInfo {
```

Comment:
This struct essentially holds the delta between two location updates. Let’s call the property `lastQualityLocationInterval` and the struct `LocationInterval`.